tools/hardware/utils/cpu.py

```diff
@@ -16,6 +16,8 @@
 def get_cpus(tree: FdtParser) -> List[WrappedNode]:
     ' Return a list of all the CPUs described in this device tree. '
     cpus_node = tree.get_path('/cpus')
+    if cpus_node is None:
+        return []
```

Comment:
How about this idiom?

```suggestion
    cpus_node = tree.get_path('/cpus') or []
```